### PR TITLE
access-group developed

### PR DESCRIPTION
**Pull request by poornima-metron**

### CODE REVIEW

**Review by suraj-metron — CHANGES_REQUESTED** [review]

**Review by suraj-metron — APPROVED** [review]

File: src/steps/active-directory/index.ts [hunk 1]
```diff
@@ -105,21 +105,33 @@ export async function fetchUsers(
...
-      user,
-      userRegistrationDetailsMap?.get(user.id as string),
-    );
-    await jobState.addEntity(userEntity);
-    await jobState.addRelationship(
-      createAccountUserRelationship(accountEntity, userEntity),
-    );
+    const userId = user.id as string;
+    await graphClient.fetchUserDetails(userId, async (userDetials) => {
+      details = userDetials;
```
Comment (Bharatkk-metron) [inline src/steps/active-directory/index.ts:116]: Do we need a `details` array? can't we directly get userDetails[0], userDetails[1], userDetails[2], userDetails[3]?

Comment (poornima-metron) [reply src/steps/active-directory/index.ts]: fixed this , using userDetails

File: src/index.ts [hunk 2]
```diff
@@ -115,6 +116,8 @@ export const invocationConfig: IntegrationInvocationConfig<IntegrationConfig> =
       ...eventHubStep,
       ...expressRouteSteps,
       ...applicationSecurityGroupSteps,
+      ...accessPackageSteps
+
```
[the remark this answers is not in the document]
Comment (poornima-metron) [reply src/index.ts:80]: removed

**Review by Bharatkk-metron — APPROVED** [review]
Looks good to me.

**Pull request merged**